### PR TITLE
Fix Account Store logs & Add constant for genesis accounts

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
## Description
* Remove redundant FIXME and LOG
* Add constant TOTAL_GENESIS_TOKEN for distribution to genesis accounts.
* Change the hard-coded pubkey used for Genesis DS block

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test (commit: e585c829)

- [ ] ~small-scale cloud test~

### CODE REVIEW

File: constants.xml [hunk 1]
```diff
@@ -208,6 +208,7 @@
     <transactions>
         <TOTAL_COINBASE_REWARD>8400000000000000000000</TOTAL_COINBASE_REWARD>
         <COINBASE_REWARD_PER_DS>191780820000000000</COINBASE_REWARD_PER_DS>
+        <TOTAL_GENESIS_TOKEN>1000000</TOTAL_GENESIS_TOKEN>
```
Comment (jiayaoqijia) [inline constants.xml]: Please change it to 12600000000000000000000.

File: constants_local.xml [hunk 2]
```diff
@@ -208,6 +208,7 @@
     <transactions>
         <TOTAL_COINBASE_REWARD>8400000000000000000000</TOTAL_COINBASE_REWARD>
         <COINBASE_REWARD_PER_DS>191780820000000000</COINBASE_REWARD_PER_DS>
+        <TOTAL_GENESIS_TOKEN>1000000</TOTAL_GENESIS_TOKEN>
```
Comment (jiayaoqijia) [inline constants_local.xml]: 12600000000000000000000

File: constants_local.xml [hunk 3]
```diff
@@ -208,6 +209,7 @@
     <transactions>
         <TOTAL_COINBASE_REWARD>8400000000000000000000</TOTAL_COINBASE_REWARD>
         <COINBASE_REWARD_PER_DS>191780820000000000</COINBASE_REWARD_PER_DS>
+        <TOTAL_GENESIS_TOKEN>12600000000000000000000</TOTAL_GENESIS_TOKEN>
```
Comment (bb111189) [inline constants_local.xml]: shall we add xml comment saying the unit is in `Qa` and not `zil` to avoid confusion

File: src/libNode/Node.cpp [hunk 4]
```diff
@@ -70,15 +70,25 @@ const unsigned int MIN_CHILD_CLUSTER_SIZE = 2;
...
+  bool moduloCredited = false;
 
   for (auto& walletHexStr : GENESIS_WALLETS) {
     bytes addrBytes;
     if (!DataConversion::HexStrToUint8Vec(walletHexStr, addrBytes)) {
       continue;
     }
     Address addr{addrBytes};
+    uint128_t bal = 0;
+    if (!moduloCredited) {
```
Comment (bb111189) [inline src/libNode/Node.cpp]: What is this logic doing?

File: constants_local.xml (hunk 3 above)
Comment (kaikawaliu) [reply constants_local.xml]: I think all of our unit for balance in protocol side are `Qa` intentionally

File: src/libNode/Node.cpp (hunk 4 above)
Comment (kaikawaliu) [reply src/libNode/Node.cpp]: I think he is trying to put more balance that left by dividing to the first genesis account

**Pull request merged**